src/app/profiles/store/reducers/connection.reducer.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ConnectionAction,
+  CONNECT,
+  CONNECTED,
+  DISCONNECTED
+} from 'app/store/actions/connection';
+
+export type ConnectionState = {
+  connecting: number;
```

Comment:
I don’t understand why `connecting` is a number and not a `boolean` ?

Reply:
As for the `fetching` flag of the collection, for example, what if several `connect` are launch in parallel? Why might see we are connected whereas not all call have returned... think about it, there are many cases where it's more consistent to wait for all connection calls to return before telling we are not attempting to connect anymore.

Reply:
Parallel fetch of the same collection ?